package.json

```diff
@@ -51,6 +51,7 @@
   },
   "dependencies": {
     "archiver": "~1.3.0",
+    "babel-runtime": "^6.23.0",
```

Comment:
Not sure if transform-runtime is even necessary? You need babel-runtime to be a dependency - it must already work because of npm 3?